pkg/operator/apis/config/v1alpha1/types.go

```diff
@@ -92,6 +95,18 @@ type Server struct {
 	Port int `json:"port"`
 }
 
+// NodeTolerationConfiguration contains information about node toleration options.
+type NodeTolerationConfiguration struct {
+	// DefaultNotReadyTolerationSeconds specifies the seconds for the `node.kubernetes.io/not-ready` toleration that
+	// should be added to pods not already tolerating this taint.
+	// +optional
+	DefaultNotReadyTolerationSeconds *int64 `json:"defaultNotReadyTolerationSeconds,omitempty"`
+	// DefaultUnreachableTolerationSeconds specifies the seconds for the `node.kubernetes.io/unreachable` toleration that
+	// should be added to pods not already tolerating this taint.
+	// +optional
+	DefaultUnreachableTolerationSeconds *int64 `json:"defaultUnreachableTolerationSeconds,omitempty"`
+}
```

Comment:
Shouldn't we add defaulting for these fields (to `60s`)?

Reply:
I had added this logic but then removed it again because having defaults for both fields didn't allow to only configure one of the tolerations, e.g.:

```
nodeToleration:
  defaultNotReadyTolerationSeconds: 120
```

It's probably not a common use case but think we should still support it.

Reply:
OK